### PR TITLE
refactors e2e templates

### PR DESCRIPTION
**Pull request by xh3b4sd**
Towards https://github.com/giantswarm/giantswarm/issues/4199.

### CODE REVIEW

File: helm/kvm-operator-chart/templates/configmap.yaml [hunk 1]
```diff
@@ -13,7 +13,7 @@ data:
         address: 'http://0.0.0.0:8000'
     service:
       crd:
-        labelSelector: '{{ .Values.labelSelector }}'
+        labelSelector: '{{ .Values.Installation.V1.GiantSwarm.KVMOperator.LabelSelector }}'
```
Comment (xh3b4sd) [inline helm/kvm-operator-chart/templates/configmap.yaml]: Using more proper installation values here. See also https://github.com/giantswarm/installations/pull/487.

File: integration/template/kvm_operator_charts_values.go [hunk 2]
```diff
@@ -5,6 +5,9 @@ package template
 // variables will be expanded before writing the contents to a file.
 var KVMOperatorChartValues = `Installation:
   V1:
+    GiantSwarm:
+      KVMOperator:
+        LabelSelector: 'giantswarm.io/cluster=${CLUSTER_NAME}'
```
Comment (xh3b4sd) [inline integration/template/kvm_operator_charts_values.go]: Aligns with the installation values. Note that I also changed the label selector according to the new cluster ID label. Before we used the deprecated one. Should be good due to https://github.com/giantswarm/certs/blob/master/k8s.go#L70-L77.

File: helm/kvm-operator-chart/values.yaml [hunk 3]
```diff
@@ -1,2 +1 @@
-labelSelector: ''
```
Comment (xh3b4sd) [inline helm/kvm-operator-chart/values.yaml]: That should now not be needed anymore.

**Review by xh3b4sd — COMMENTED** [review]
Note that this is only the first step. In the next step I want to move the template from this repo to `e2e-templates`. And then I want to go with the newer approach we have there.

**Review by MarcelMue — APPROVED** [review]
LGTM :+1:

**Review by rossf7 — APPROVED** [review]

**Review by calvix — APPROVED** [review]

**Review by kopiczko — APPROVED** [review]

**Review by calvix — APPROVED** [review]

**Pull request merged**